llvm/lib/Passes/PassRegistry.def

```diff
@@ -164,6 +164,7 @@ MODULE_PASS("lower-slm-reservation-calls", ESIMDLowerSLMReservationCalls())
 MODULE_PASS("record-sycl-aspect-names", RecordSYCLAspectNamesPass())
 MODULE_PASS("sycl-virtual-functions-analysis",
             SYCLVirtualFunctionsAnalysisPass())
+MODULE_PASS("spec-constants", SpecConstantsPass())
```

Comment:
Thanks! Most of pass tests use incorrect approach with testing it through `sycl-post-link`, this is a way better direction